CHANGELOG.md

```diff
@@ -28,6 +28,17 @@ which it is deprecated.
   property now. The value is calculated based on current market conditions to prevent
   refunds while protecting against big price movements. It can be passed into
   the `requestDepositAddressV2` method to set the slippage tolerance for a swap.
+- Optional parameter to enable recommended slippage during SDK initialization. The default value
```

Comment:
think i would not document this for now and enable it by default (remove the option) once we are confident. but not a blocker for me